specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ServerConfigurationOptions.json

```diff
@@ -0,0 +1,264 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets a list of managed instance server configuration options.": {
+            "$ref": "./examples/ServerConfigurationOptionList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/serverConfigurationOptions/{serverConfigurationOptionName}": {
+      "get": {
+        "tags": [
+          "ServerConfigurationOptions"
+        ],
+        "description": "Gets managed instance server configuration option.",
+        "operationId": "ServerConfigurationOptions_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "serverConfigurationOptionName",
+            "in": "path",
+            "description": "The name of the server configuration option.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "allowPolybaseExport"
+            ],
+            "x-ms-enum": {
+              "name": "ServerConfigurationOptionName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified managed instance server configuration option.",
+            "schema": {
+              "$ref": "#/definitions/ServerConfigurationOption"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidServerConfigurationOptionName - User specified server configuration option name that is not supported.\n\n * 400 InvalidServerConfigurationOptionValue - Server configuration option value \"{0}\" is not supported for server configuration option \"{1}\".\n\n * 400 InvalidServerConfigurationOptionProperties - The provided set of parameters in the body of the request is either empty or invalid.\n\n * 404 SubscriptionNotFound - The requested subscription was not found.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Gets managed instance server configuration option.": {
+            "$ref": "./examples/ServerConfigurationOptionGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServerConfigurationOptions"
+        ],
+        "description": "Updates managed instance server configuration option.",
```

Comment:
>Updates [](http://example.com/codeflow?start=24&length=7)

Only update and not create? Who creates it?

Reply:
The configuration option which is updated via PUT request is one of sys.configurations options (currently only allow polybase export option is supported) which already exist within SQL engine. So user cannot create or delete the config, they will be only able to update it.